HealthcareSystem/Backend/Model/Users/DoctorSpecialty.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Model.Users;
+
+namespace Backend.Model.Users
+{
+    public class DoctorSpecialty
+    {
+        public string DoctorJmbg { get; set; }
+        public virtual Doctor Doctor { get; set; }
+        public int SpecialtyId { get; set; }
+        public virtual Specialty Specialty { get; set; }
+    }
```

Comment:
I don't think you actually need to have this class in the object model. Look at [this](https://www.entityframeworktutorial.net/code-first/configure-many-to-many-relationship-in-code-first.aspx), Entity Framework should be capable of generating a table for the relationship on it's own. So it should be enough to just have a virtual list of Specialty objects in Doctor, and a virtual list of Doctor objects in Specialty. I feel like that would be more convenient to use.

Reply:
Correction: this is apparently a pretty new feature in EF Core 5.0, and not available in EF Core 3.1 which we are using.